### ISSUE TITLE
domain-resolv-preview not compiling

### ISSUE THREAD

**Issue by pusateri** [NLnetLabs/domain-core#1]
```
   Compiling domain-resolv-preview v0.3.1 (https://github.com/NLnetLabs/domain-core.git#c0463de8)
error[E0432]: unresolved import `futures_util::compat::TokioDefaultSpawn`       
  --> /Users/pusateri/.cargo/git/checkouts/domain-core-5d0a0b48f993e038/c0463de/domain-resolv/src/resolver.rs:16:5
   |                                                                            
16 | use futures_util::compat::TokioDefaultSpawn;                               
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `TokioDefaultSpawn` in `compat`. Did you mean to use `TokioDefaultSpawner`?
                                                                                
error[E0061]: this function takes 0 parameters but 1 parameter was supplied     
   --> /Users/pusateri/.cargo/git/checkouts/domain-core-5d0a0b48f993e038/c0463de/domain-resolv/src/resolver.rs:135:44
    |                                                                           
135 |         op(&resolver).boxed().unit_error().compat(TokioDefaultSpawn)      
    |                                            ^^^^^^ expected 0 parameters   
                                                                                
error: aborting due to 2 previous errors                                        
                                                                                
Some errors occurred: E0061, E0432.                                             
For more information about an error, try `rustc --explain E0061`.               
error: Could not compile `domain-resolv-preview`.
```

**Comment by pusateri** [comment]
Using the following in my cargo.toml:
```
[dependencies]
domain-core = { git = "https://github.com/NLnetLabs/domain-core.git" }
domain-resolv-preview = { git = "https://github.com/NLnetLabs/domain-core.git" }
```

**Comment by partim** [comment]
Sorry for the delay. I just now pushed the initial version of the re-written resolver.

I need to figure out a good way to do some testing of it and then should probably make a release …

**Comment by pusateri** [comment]
Fixed in 49bdc95b533b0e54c4618cc1eef2c8f4e179e616
